index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Product landing page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
+    <header id="header">
+      <nav id="nav-bar">
+          <img id="header-img" class="logo" src="/samsung.png" alt="samsung-logo">
+          <ul class="nav-menu">
```

Comment:
You have done very well on this HTML document, but your code's randomly placed blank spaces make it a bit troublesome to read. I suggest you keep it consistent with where you place your blank lines. For instance, you can space out sibling elements with blank lines to make your code more readable instead of placing them right after opening tags.